client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningInfo.tsx

```diff
@@ -46,19 +46,21 @@ const ContactQuestioningInfo: React.FC<Props> = (props: Props): JSX.Element => {
                 confirmButtonText: 'כן, המשך'
             }).then((result) => {
                 if (result.value) {
-                    let foundDuplicateIds: boolean = false;
                     if (interactedContact.identificationNumber !== '') {
-                        axios.post('/contactedPeople/checkDuplicates', {currIdNumber: interactedContact.identificationNumber})
+                        axios.post('/contactedPeople/checkDuplicates', {
+                            currIdNumber: interactedContact.identificationNumber,
+                            personId: interactedContact.id
```

Comment:
maybe use a more readable name..
the difference between currIdNumber and personId isnt very clear?

Reply:
fixed